### ISSUE TITLE
Add support for Windows bat (batch) files

### ISSUE THREAD

**Issue by AndiDittrich** [EnlighterJS/EnlighterJS#186]
https://wordpress.org/support/topic/what-language-can-be-used-for-batch-file-or-bash-script-highlighting/#post-15304027

**Comment by gsool9988** [comment]
How can I apply the new batch.js to my wordpress system?

I copied all files in /dist folder and pasted & overwrited to the folder,  "[my-wordpress-plugin-folder]\enlighter\resources\enlighterjs\".
Then, I cleared out the cache folder, "[my-wordpress-plugin-folder]\enlighter\cache\".
But no effect. 
I tried rebooting my local wordpress server and local computer.
But "batch" language is not appeared on enlighterJS code insertor in wordpress editor.
Did I missed something in the steps?

[My system & enlighter plugin information]
    Server
    PHP Version: 7.4.27
    PHP JSON Extension: available
    Server Software: WINNT

    Components
    Enlighter Plugin: 4.4.2 CE
    EnlighterJS Version: 3.6.0
    TinyMCE Extension: 5.0.1
    Gutenberg Extension: 1.1.1
    Theme Customizer: 2.3.0
    WP-Skeleton Version: 0.28.0
